src/visualization/adapt.jl

```diff
@@ -0,0 +1,58 @@
+"""
+    adapt_to_mesh_level!(u_ode::AbstractVector, semi, level)
+    adapt_to_mesh_level!(sol::Union{DiffEqBase.ODESolution,TimeIntegratorSolution}, level)
```

Comment:
```suggestion
    adapt_to_mesh_level!(sol::Trixi.TrixiODESolution, level)
```
We do not export/explain `TimeIntegratorSolution` and this does not work with general `DiffEqBase.ODESolution`s.

Reply:
Thanks!